include/vast/Util/Scopes.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021-present, Trail of Bits, Inc.
+
+#pragma once
+
+#include <vast/Util/Common.hpp>
+
+namespace vast
+{
+    inline bool is_trailing_scope(operation op) {
```

Comment:
```suggestion
    static inline bool is_trailing_scope(operation op) {
```
And same goes for the one below.